### DISCUSSION TITLE
Rus ruleti ve olasılık sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#34] [Zeka Sorusu] [answered]
İki kişi, bir altıpatlar tabanca ile Rus ruleti oynamaya karar veriyorlar.
Bir tanesi, diğerinin de görebilecegi şekilde tabancaya YANYANA 2 gerçek mermi koyuyor. Sonra tabancanın topunu hızla çevirip mermilerin yerinin unutulmasını sağlıyor. Silahı kafasına dayayıp tetiğe basıyor. Silah patlamıyor. Tetiğe basıldığı için mermileri tutan top bir sıra ilerliyor.
Silahı karşısındakine veriyor.
Soru: Sıra kendisine gelen adam, tetiği çekmeden önce tabancanın topunu hızla çevirmeli midir, çevirmemeli midir? Niye?

**Comment by AcelyaP** [comment, marked as answer]
Çevirmemesi lazım hemen şekillerle anlatayım. Altıpatlar tabancadaki mermi şekli aşağıdaki gibidir.
-: boş  +: dolu
++----
+----+
----++
---++-
--++--
-++---
İlk elde ateş etmediğine göre 1 ve 2. seçeneği eliyoruz. Her çevirmede bir ilerlediği için aşağıdaki gibi olur.
---++-
--++--
-++---
++----
tek 4. seçenekte ölürüz o yüzden 1/4 olasılıkladır.
